synr/ast.py

```diff
@@ -595,7 +595,7 @@ class For(Stmt):
     and :code:`body` will be :code:`pass`.
     """
 
-    lhs: Var
+    lhs: Expr
```

Comment:
I don't think we want an arbitrary expression here. I think we want a list of vars. @jroesch do you have input?